angle-ui/src/Pages/Login/login.css

```diff
@@ -10,4 +12,93 @@
 
 .signup-group {
     margin: 2%;
+    
+}
+
+h1 {
+    font-weight: bold;
+    font-size: 30px;
+}
+
+.form-container {
+	position: absolute;
+	top: 0;
+	height: 100%;
+}
+
+.sign-in-container {
+	left: 0;
+	width: 50%;
+	z-index: 2;
+}
+
+
+form {
+	background-color: #FFFFFF;
+	display: flex;
+	align-items: center;
+	flex-direction: column;
+	padding: 50px 50px;
+	height: 50%;
```

Comment:
When using px and % together you want to be careful how it will look on different displays. I am not too sure what to do here. Maybe @nanarboursal can help.